Source/SourceKittenFramework/Clang+SourceKitten.swift

```diff
@@ -194,7 +194,7 @@ extension CXCursor {
         return commentBody
     }
 
-    func swiftDeclaration(compilerArguments: [String]) -> String? {
+    func swiftDeclarationAndName(compilerArguments: [String]) -> (String?, String?) {
```

Comment:
should this return a tuple with named elements?

Reply:
Done